201-timeseriesinsights-environment-with-eventhub/azuredeploy.json

```diff
@@ -91,6 +91,20 @@
       "metadata": {
         "description": "The name of the shared access key that the Time Series Insights service will use to connect to the event hub."
       }
+    },
+    "accessPolicyReaderObjectIds": {
+      "type": "array",
+      "defaultValue": ["thisArrayIsReallyEmpty"],
```

Comment:
just use an empty array then... e.g.

"defaultValue": [ ],

Reply:
I started with that, but the copy seems unable to enumerate over an empty collection:
>Deployment template validation failed: 'The template 'copy' definition at line '223' and >column '20' has an invalid copy count. The copy count must be a postive integer value and cannot exceed '800'.
This was the workaround I was able to come up with - because of the condition on the access policy resources, the copy never gets evaluated if the array is empty.

Reply:
I suppose an alternative is to conditionally set the copy.count to 1 when the array is empty. But that feels like just as much a hack and one that I can't document in the template. I thought the default value I picked would help make the reason for the workaround more apparent.